gw_eccentricity/load_data.py

```diff
@@ -518,26 +518,38 @@ def get_defaults_for_nr():
 def load_lvcnr_waveform(**kwargs):
     """Load modes from lvcnr files.
 
+    Loading waveform modes from files in lvcnr format require the file
+    `SEOBNRv4ROM_v2.0.hdf5` in `LAL_DATA_PATH`.  NR simulation data is
+    compressed using ROM and saved in LVCNR format to reduce file size and the
+    above file is required to correctly interpolate the NR data while loading
+    the waveform modes.
```

Comment:
Not quite right, I think it's also used to get the peak time for some reason.
Just simplify and say the same thing you say for zeroecc: Needs so and so file, get it from here, set LAL_DATA_PATH, etc.

Reply:
Done